udata/core/dataset/event.py

```diff
@@ -0,0 +1,58 @@
+import logging
+from uuid import UUID
+
+from .models import get_resource
+log = logging.getLogger(__name__)
+
+
+def consume_message_resource_analysed(key, value):
+    '''
+    Reads a message and update the resource extras with the payload values.
```

Comment:
The three docstrings are the same could it be more explicits? The difference between the threee functions was not really straightforward to me 😬